src/status_im/contexts/wallet/signals.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im.contexts.wallet.signals
+  (:require
+    [taoensso.timbre :as log]
+    [utils.re-frame :as rf]))
+
+(rf/reg-event-fx
+ :wallet/signal-fired
+ (fn [_ {:keys [type blockNumber accounts] :as event}]
+   (log/debug "[wallet-subs] new-wallet-event"
+              "event-type"  type
+              "blockNumber" blockNumber
+              "accounts"    accounts)
+   (case type
+     "wallet-owned-collectibles-filtering-done" {:fx [[:dispatch
```

Comment:
I like the structure you guys are applying to isolate wallet signals within the `wallet` context 👍🏼 Good example to follow.